src/hosting/functionsProxy.ts

```diff
@@ -49,6 +49,6 @@ export default function(
       }
     }
 
-    return await proxyRequestHandler(url, `${destLabel} Function ${rewrite.function}`);
+    return Promise.resolve(proxyRequestHandler(url, `${destLabel} Function ${rewrite.function}`));
```

Comment:
Why this change?

Reply:
Two fold:
1. proxyRequestHandler is _not_ async, and does _not_ return a promise (async and await are removed)
2. Since the signature of the function required a promise (and the JSDocs too), the returned handler must be wrapped in a promise.